websites/O/osu!track/presence.ts

```diff
@@ -0,0 +1,87 @@
+const presence = new Presence({
+    clientId: "861180231909113866"
+  }),
+  timestampe = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+  const presenceData: PresenceData = {
+    largeImageKey: "logo",
+    startTimestamp: timestampe
+  },
+    pathnames = location.pathname,
+    urlSplit = document.URL.split("/"),
```

Comment:
Seems like you always use [5], maybe use array deconstructing?

Reply:
i forgot how to do it but ill try

Reply:
[, , , , urlSplit] = document.URL.split("/")

Reply:
figured it out

Reply:
i did 
`urlSplitRaw = document.URL.split("/"),
[, , , , , urlSplit] = urlSplitRaw,`

worked for me